.github/publish-to-pypi.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish Python distributions to PyPI and TestPyPI
+
+on: push
+
+jobs:
+  build-and-publish:
+    name: Build and Publish
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.9
+    - name: Install pypa/build
+      run: >-
+          python -m
+          pip install
+          build
+          --user
+    - name: Build source tarball
+      run: >-
+          python -m
+          build
+          --sdist
+          --outdir dist/
+          .
+#   - name: Publish distribution to Test PyPI
+#     uses: pypa/gh-action-pypi-publish@master
+#     with:
+#       password: ${{ secrets.TEST_PYPI_API_TOKEN }}
+#       repository_url: https://test.pypi.org/legacy/
+    - name: Publish distribution to PyPI
+      if: startsWith(github.ref, 'refs/tags')
+      uses: pypa/gh-action-pypi-publish@master
```

Comment:
probably not a good idea to use the master branch for any given action. If they provide clearly labelled version we should use that instead.